README.md

```diff
@@ -1139,6 +1172,7 @@ Valid keys and their meanings:
     * **final_workflow_outputs_dir** - Specifies a path where final workflow outputs will be written.  If this is not specified, workflow outputs will not be copied out of the Cromwell workflow execution directory/path.
     * **final_call_logs_dir** - Specifies a path where final call logs will be written.  If this is not specified, call logs will not be copied out of the Cromwell workflow execution directory/path.
     * **default_runtime_attributes** - A JSON object where the keys are [runtime attributes](#runtime-attributes) and the values are defaults that will be used through the workflow invocation.  Individual tasks can choose to override these values.  See the [runtime attributes](#specifying-default-values) section for more information.
+    * **continueOnReturnCode** - Can accept true or false.  If false, then only return code of 0 will be acceptable for a task invocation.  If true, then any return code is valid.  This can also accept a list of integers which enumerate the acceptable return codes.
```

Comment:
Could you also mention the order of precedence? Presumably this is a default which individual tasks' runtime attrs can override?